baybe/recommenders/pure/bayesian/botorch.py

```diff
@@ -0,0 +1,281 @@
+"""Botorch recommender."""
```

Comment:
because of the file move its hard to see the differences, I see the `sequential` flag addition (expected), is there nay other thing that was changed worth mentioning?

Reply:
Not really, apart from a slight reformatting of the docstring. But note: I did the movement in a single commit (on purpose) so that you can very easily see the diff if you compare not against the root but against the first commit of this PR. I always do it like that so that people have an easier time reviewing. Let me know if you have more questions or close 🙃